module/module.go

```diff
@@ -802,12 +790,9 @@ func (m *Module) RemoveStream(ctx context.Context, req *streampb.RemoveStreamReq
 		return nil, errors.Errorf("stream %s is not active", name)
 	}
 
-	if err := m.pc.RemoveTrack(prs.sender); err != nil {
```

Comment:
No longer needed as `Unsubscribe` must cancel the `Subscription` Context, which will trigger https://github.com/viamrobotics/rdk/pull/3792/files#diff-9a0a92d26bcd93d88bcfe6d94c11871002ddb11ea827c5f27c75c3238464d079R745